src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsPrefooters.java

```diff
@@ -14,9 +15,14 @@
       dataProvider = "adsMiddlePrefooter",
       groups = "AdsMiddlePrefooter"
   )
-  public void adsMiddlePrefooter(String wikiName, String path, boolean middlePrefooterEnabled) {
+  public void adsMiddlePrefooter(
+      String wikiName,
+      String path,
+      Dimension windowResolution,
+      boolean middlePrefooterEnabled
+  ) {
     String testedPage = urlBuilder.getUrlForPath(wikiName, path);
-    AdsBaseObject adsBaseObject = new AdsBaseObject(driver, testedPage);
+    AdsBaseObject adsBaseObject = new AdsBaseObject(driver, testedPage, windowResolution);
```

Comment:
Thanks @justnpT! Yeah, I've seen it running the changed test on my local machine. Chrome was able to display it in a window higher than my screen size whereas Firefox opened it just in the maximized window. Both solution were fine for this test and make it stable. However, screenshot comparison results would fail here and this is something we should think about if we decide to go with some common repository-wide approach. I agree! :tada: